apps/documentation/src/routes/documentation/routing/intro.mdx

```diff
@@ -1,16 +1,18 @@
-import { Head } from 'tuono'
+import MetaTags from '../../../components/meta-tags'
 
-<Head>
-  <title>Tuono - Routing</title>
-  <link rel="canonical" href="https://tuono.dev/documentation/routing/intro" />
-</Head>
+<MetaTags
+  title="Tuono - Routing"
+  canonical="https://tuono.dev/documentation/routing"
+/>
 
 import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
-<Breadcrumbs>
-  <Element label="Routing" href="/documentation/routing" />
-  <Element label="Project structure" />
-</Breadcrumbs>
+<Breadcrumbs
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Possible typo: you repeated a word (ENGLISH_WORD_REPEAT_RULE)
Suggestions: `Breadcrumbs`
Rule: https://community.languagetool.org/rule/show/ENGLISH_WORD_REPEAT_RULE?lang=en-US
Category: MISC
<!-- __reviewdog__:ChA1NjQzNDdmZDA3OWNiOTc0EgxMYW5ndWFnZVRvb2w= -->